### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by tharvik**
A bit more information for newcomers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+[![codecov](https://codecov.io/gh/Distributed-EPFL/at2-node/branch/master/graph/badge.svg)](https://codecov.io/gh/Distributed-EPFL/at2-node)
+
+An implementation of a distributed ledger using
+[AT2](https://arxiv.org/abs/1812.10844) (Asynchronous Trustworthy Transfers).
+It allows to send money to a chosen identity provided that you have enough in
+your account, while being byzantine resistant.
+
+## overview
+
+There is two binaries, one for the server and one for the client, both in rust.
```
Comment (SonicFrog) [inline README.md]:
```suggestion
There are two binaries, one for the server and one for the client, both in rust.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# at2-node
+
+[![at2-node](https://github.com/Distributed-EPFL/at2-node/actions/workflows/rust.yml/badge.svg)](https://github.com/Distributed-EPFL/at2-node/actions/workflows/rust.yml)
+[![codecov](https://codecov.io/gh/Distributed-EPFL/at2-node/branch/master/graph/badge.svg)](https://codecov.io/gh/Distributed-EPFL/at2-node)
+
+An implementation of a distributed ledger using
+[AT2](https://arxiv.org/abs/1812.10844) (Asynchronous Trustworthy Transfers).
+It allows to send money to a chosen identity provided that you have enough in
```
Comment (ineiti) [inline README.md]:
```suggestion
It allows to send tokens to a chosen identity provided that you have enough in
```

Comment (SonicFrog) [reply README.md:8]: Also allow is not the proper verb here, a better phrasing would be something along the lines of `It provides an easy way to transfer tokens from identity to identity provided there are enough in the source identity's account`

Comment (SonicFrog) [inline README.md]:
```suggestion
It allows you to send money to a chosen identity provided that you have enough in
```

Comment (tharvik) [reply README.md]: Hum, I'm using "money" throughout the code, even in the CLI. Keeping it as it makes it clearer IMO

Comment (ineiti) [reply README.md]: Don't use `money`. It should be either one of `token`, `asset`, or `coin`. Even if I think the latter is also too much noted into fiat money. Following the name of the project, it should probably be `asset`. `Token` is probably better for the NonFungibleTokens...

Comment (tharvik) [reply README.md]: Alright, "asset" it is!

**Comment by tharvik** [comment]
@SonicFrog quite bothering this codecov issue, you can disregard it.

**Comment by tharvik** [comment]
Sooo, I've been able to finally make `kcov` works with subprocess :D
And disabled windows build (because of `nix`) and send some `SIGTERM` before killing a server so that kcov can still save state and re-up the codecov limits.

**Comment by tharvik** [comment]
@SonicFrog so, the patch itself is not really hiting the threshold, as I've added a kcov-helper (which can't be covered), so yeah, you can merge it

**Review by SonicFrog — APPROVED** [review]

**Pull request merged**